charts/pravega-operator/templates/certificate.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Issuer
+metadata:
+  name: selfsigned-cert
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: pravega-operator
+    app.kubernetes.io/version: "{{ .Chart.AppVersion }}"
```

Comment:
the appVersion has been removed from Chart.yaml since it requires separate maintenance. In that case the version can be picked from `image.tag`. What do u think?

Reply:
Out of curiosity, is it much more work to change the string in both the `appVersion` and `image.tag` when we do versioning bumps in this repo?

From what I can see, `app.kubernetes.io/version: "{{ .Chart.AppVersion }}"` is the best practice: https://helm.sh/docs/chart_best_practices/labels/ (see the "Standard Labels" section)

Reply:
`appVersion` is also what appears when you do `helm list`